README.md

```diff
@@ -16,15 +17,16 @@ Kili is a platform that empowers a data-centric approach to Machine Learning thr
 
 and [many more](https://cloud.kili-technology.com/docs/labeling/labeling-overview/#available-labeling-job-types-per-asset-type-a-idjob-types-per-asset-typea).
 
-## What is Kili Playground?
-
-Kili Playground is the home of the **Kili Python SDK**. It comes with several [recipes](recipes/) that demonstrate its most frequent use cases.
+## What is Kili Python SDK?
 
+**Kili Python SDK** is the Software Development Kit of Kili. It allows to automate actions on the platform, query and mutate data and provide a better experience. It comes with several [recipes](recipes/) that demonstrate its most frequent use cases.
```

Comment:
**Kili Python SDK** is the Python client for the Kili platform. It allows to query and manipulate the main entities available in Kili, like projects, assets, labels, api keys... It comes with several [recipes](recipes/) that demonstrate how to use it in the most frequent use cases.